core/src/main/java/hudson/cli/CLIAction.java

```diff
@@ -118,12 +118,26 @@ public boolean isWebSocketSupported() {
         return WebSockets.isSupported();
     }
 
+    /**
+     * Unlike {@link HttpResponses#errorWithoutStack} this sends the message in a header rather than the body.
+     * (Currently the WebSocket CLI is unable to process the body in an error message.)
+     */
+    private static HttpResponse statusWithExplanation(int code, String errorMessage) {
+        return new HttpResponse() {
+            @Override
+            public void generateResponse(StaplerRequest2 req, StaplerResponse2 rsp, Object node) {
```

Comment:
Note that if anyone had been using this as a `@FunctionalInterface` before, that would break as of https://github.com/jenkinsci/stapler/pull/482. Not sure if this was only source-incompatible rather than binary-incompatible? https://docs.oracle.com/javase/specs/jls/se23/html/jls-13.html#jls-13.5.4 does not seem to mention SAMs.